packages/epics/src/agreements/validation/agreement-form-payment-schedule-validation.ts

```diff
@@ -0,0 +1,47 @@
+import { isBefore } from 'date-fns';
+
+type DateRange = {
+  from: Date | undefined;
+  to: Date | undefined;
+};
+
+type Milestone = {
+  percentage: string;
```

Comment:
@n0umen0n, what should the data type for `Milestone.precentage` be?

Reply:
Since I don't need it on the contract side, whatever is more convenient on your side. Once the milestone is reached and user creates request for payment you might use this value to calculate the amount?

Reply:
@plitzenberger I converted it to number